### DISCUSSION TITLE
Can somebody help me? ZBmini not supported

### DISCUSSION THREAD

**Discussion by Phrana** [Koenkk/zigbee2mqtt#15274] [Devices]
Hello, I have HA and Zigbee2MQTT installed, it detects the SNZB-03, SNZB-04, but the ZBmini tells me that it does not recognize it.
In the registry it puts this:
Error 2022-12-01 14:05:24Failed to execute LQI for 'Conmutador entrada'
Thanks

**Comment by Phrana** [comment]
Solved

**Reply by mikip4** [reply]
How did you solved ?  same problem

**Reply by Phrana** [reply]
Hello, you have to bring it to the coordinator

**Reply by mikip4** [reply]
Thanks for replay, how can í do It?

**Comment by Phrana** [comment]
Well, you have two options, either bring the coordinator closer to the switch where it is connected, or bring the switch closer to the coordinator and connect it near the coordinator

**Reply by mikip4** [reply]
ok, thanks, it was the distance

**Comment by Phrana** [comment]
Ok I'm glad
